### DISCUSSION TITLE
Upgrade to SQL 2016 SP3

### DISCUSSION THREAD

**Discussion by ohlssrog** [dataplat/dbatools#7800] [Q&A] [answered]
Hi. I wanna upgrade my sql 2016 SP2 servers to latest SP3, when i run the command like

Update-DbaInstance -ComputerName sqlserv01 -Type ServicePack -Path \\networkpath\share\SQLCumulativeUpdates\SQL2016 -Restart -Credential $cred

Sql server is upgraded with latest CU17 and  have the Product ID 13.0.5888.11, and that seems to be a problem to install SP3, i get the output 
WARNING: [13:48:01][Get-SqlInstanceUpdate] 13.0.5888 on computer [sqlserv01] is already the latest available.

How  can i upgrade my SQL 2016 to ltest SP3 with DBA Tools and Update-DbaInstance

**Comment by ohlssrog** [comment, marked as answer]
After i update-Module dbatools and run the update-dbabuildref it all starts to work :)

**Reply by niphlod** [reply]
it would have sufficed a run with Update-DbaBuildReference but updating the whole module works, too.
